crates/ghost_actor/src/ghost_actor.rs

```diff
@@ -226,12 +226,12 @@ pub struct GhostParentWrapperDyn<
 }
 
 impl<
-        Context,
+        Context: 'static,
```

Comment:
The consequence of this is that we wouldn't be able to store any non-static references in context enums... which may be ok since they have access to the "user data" for any reference type stuff?

Reply:
@neonphog I'm ok with this limitation but would be interested in eliminating the static constraint if possible in the future.